File: AngularLovelyjubbly/ClientApp/src/app/shared/models/refreshtoken.ts
export interface IRefreshToken {
    refreshTokenId: string;
    userId: string;
    startDate: Date;
    endDate: Date;
    isEnabled: boolean;
}

export class RefreshToken implements IRefreshToken {

    constructor(public refreshTokenId: string, public userId: string, public startDate: Date,
        public endDate: Date, public isEnabled: boolean) {
    }
}

File: AngularLovelyjubbly.Data.Sql.Models/RefreshToken.cs
using System;

namespace AngularLovelyjubbly.Data.Sql.Models
{
    public class RefreshToken
    {
        public RefreshToken()
        {
        }

        public Guid RefreshTokenId { get; set; }

        //foreign key
        public string UserId { get; set; }
        //navigation property
        public virtual ApplicationUser User { get; set; }

        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }

        public bool IsEnabled { get; set; }
    }
}
